changelog/changed-use-docsplay-for-ArmError.md

```diff
@@ -0,0 +1 @@
+Changed ArmError’s documentation to use docsplay::Display and avoid repetition
```

Comment:
So, do you really want to keep this? :)

Reply:
I don’t mind either way. It was only to make GH checks happy :)

Reply:
The changelog check is not important, there's a label to pacify it :)